solutionbox/inception/datalab_solutions/inception/_util.py

```diff
@@ -35,6 +35,17 @@ def is_in_IPython():
   except ImportError:
     return False
 
+def default_project():
```

Comment:
Datalab's function is exactly "datalab.context.Context.default().project_id". I wrap it here to avoid "import datalab" everywhere.